packages/ui-causal-graph-editor/src/types.tsx

```diff
@@ -98,15 +98,16 @@ export interface CausalGraphNode {
     extras?: Record<string, any>;
     identifier?: string;
```

Comment:
Just checking, is the identifier still optional or are they now always defined in the CaiCausalGraph? If they're always there we could do a similar update to `serializeGraphNode` and always include the identifier

What about node_class? Did we make sure it's preserved as well?

Reply:
Node_class is an extra field, so it's preserved just like any other. CaiCausalGraph actually uses identifier, so it's required. I've marked them as such as well.